### ISSUE TITLE
Opening linux created ora files on a windows machine

### ISSUE THREAD

**Issue by mitvitaminen** [mypaint/mypaint#1073]
Hi, 
I just mounted an ext4 hdd with windows and when I open the ora files I created they are empty please fix someday kind regards and thx 4 this awesome application spend lots of time with it greetings Chris

**Comment by jplloyd** [comment]
Hi,

A few questions to troubleshoot this:

Which version(s) of MyPaint did you use for saving & opening the files?

Do you get this problem when opening any other type of file on the ext4
drive on windows (images, text files, etc)?

If you copy the files from the ext4 drive to the ntfs-formatted drive (the
one windows is installed on), can those copies be successfully opened?

**Comment by jplloyd** [comment]
Just so we're on the same page: Opening png/jpg files, using MyPaint(?), directly from the ext4 drive, worked fine?

First of all, could you run `mypaint-debug.exe` (same folder as `mypaint.exe` is located) and open one of the .ora files. Afterwards, copy the log messages in the terminal window and paste them, either to a [pastebin](https://pastebin.com/) (and post the link here) or directly here in the message between three backticks, like the following:

\`\`\`
\<paste goes here\>
\`\`\`

It could also be helpful if you could share one of the problematic files (preferably a small one).
If you can't share any of them, you could create a new file with just a couple of lines or some scribbles the next time you're using the linux install.

**Comment by mitvitaminen** [comment]
HI,
I did not think of opening the png in mypaint tried that and it did not work either, fiule in win fotos shows just fine, as mypaint not crashes just opens the programm with nothing shown I doubt there be a crash or debug log, but I will try that later but not right now sry =D kind regards thanks for prompt answering greetz chris

**Comment by jplloyd** [comment]
There is no hurry. Another thing to test would be to mount your ntfs drive from Linux (instead of mounting the ext4 from Windows), copy an .ora file to the ntfs drive and then boot Windows and see if _that_ file can be opened.

**Comment by jplloyd** [comment]
Hi, did you try the suggestion in my last message (or check the debug log)?

**Comment by mitvitaminen** [comment]
Hi,

sorry it took so long but the weather was too nice;)

I tried saving the files on an usb stick, but no change
when opening run in debug mode I get following error 
```python
Mypaint version: 2.0.0+git.af5225a2
System information: Windows-10-10.0.18362-SP0
Using: Python 3.8.1, GTK 3.24.13, GdkPixbuf 2.40.0, Cairo 1.16.0, GLib 2.62.1
Traceback (most recent call last):
  File "C:/Program Files/mypaint-w64/mingw64/lib/mypaint/gui/filehandling.py", line 1073, open_cb(self=<gui.filehandling.FileHandler object>, action=<Gtk.Action object at 0x000000000c55f580 (GtkAction at 0x000000000367c730)>)
                    filename = filename_to_unicode(filename)
                    self.open_file(
                        filename,
  variables: {'self.open_file': ('local', <bound method FileHandler.open_file of <gui.filehandling.FileHandler object at 0x000000000c541ac0>>), 'filename': ('local', 'D:\\myPaint\\IblisRose.ora'), 'compat_handler': (None, []), 'selector.compat_function': ('local', <gui.compatibility.CompatSelector object at 0x000000000e656670>)}
  File "C:/Program Files/mypaint-w64/mingw64/lib/mypaint/gui/filehandling.py", line 827, open_file(self=<gui.filehandling.FileHandler object>, filename=r'D:\myPaint\IblisRose.ora', **kwargs={'compat_handler': None})
            """Load a file, replacing the current working document."""
            if not self._call_doc_load_method(
                    self.doc.model.load, filename, False, **kwargs):
  variables: {'self._call_doc_load_method': ('local', <bound method FileHandler._call_doc_load_method of <gui.filehandling.FileHandler object at 0x000000000c541ac0>>), 'self.doc.model.load': ('local', <bound method Document.load of <Document nlayers=1 bbox=Rect(0, 0, 0, 0) paintonly=False>>), 'filename': ('local', 'D:\\myPaint\\IblisRose.ora'), 'kwargs': ('local', {'compat_handler': None})}
  File "C:/Program Files/mypaint-w64/mingw64/lib/mypaint/gui/filehandling.py", line 880, _call_doc_load_method(self=<gui.filehandling.FileHandler object>, method=<bound method Document.load of <Document nlayers=1 bbox=Rect(0, 0, 0, 0) paintonly=False>>, arg=r'D:\myPaint\IblisRose.ora', is_import=False, compat_handler=<function ora_compat_handler.<locals>.handler>)
            files_summary = _IOProgressUI.format_files_summary(arg)
            ioui = _IOProgressUI(self.app, op_type, files_summary)
            result = ioui.call(
  variables: {'ioui': (None, []), '_IOProgressUI': ('global', <class 'gui.filehandling._IOProgressUI'>), 'self.app': ('local', <gui.application.Application object at 0x0000000002e0f250>), 'op_type': ('local', 'load'), 'files_summary': ('local', '„IblisRose.ora"')}
  File "C:/Program Files/mypaint-w64/mingw64/lib/mypaint/gui/filehandling.py", line 249, __init__(self=<gui.filehandling._IOProgressUI object>, app=<gui.application.Application object>, op_type='load', files_summary='„IblisRose.ora"', use_statusbar=True, use_dialogs=True)
            msg = self._OP_DURATION_TEMPLATES[op_type].format(
                files_summary = files_summary,
  variables: {'msg': (None, []), 'self._OP_DURATION_TEMPLATES': ('local', {'load': '„{file_summary}“ wird geladen …', 'import': 'Importiere Ebenen aus „{files_summary}"…', 'save': '„{files_summary}“ wird gespeichert …', 'export': 'Export nach „{files_summary}“ läuft …'}), 'op_type': ('local', 'load'), 'format': ('builtin', <built-in function format>), 'files_summary': ('local', '„IblisRose.ora"')}
KeyError: 'file_summary'
````

hope this helps

**Comment by jplloyd** [comment]
Ah, that'll teach me to miss basic questions. The problem is entirely unrelated to Linux/ext4/Windows/ntfs, but is caused by a pesky invalid string in the German translation (which I let through, no excuses there).

If I had asked "Can you open files that you saved with the windows version?" we would have gotten to this sooner. The temporary solution is to switch the language to English (or another language of your choice that isn't German) in the preferences under the "Ansicht" tab.

This is already fixed in master and will be part of the 2.0.1 bugfix release, which will be released soon™.
